src/apps/config/leader.lua

```diff
@@ -421,18 +435,19 @@ function Leader:update_configuration (update_fn, verb, path, ...)
    local to_restart =
       self.support.compute_apps_to_restart_after_configuration_update (
          self.schema_name, self.current_configuration, verb, path,
-         self.current_in_place_dependencies, ...)
+         self.in_place_dependencies, ...)
    local new_config = update_fn(self.current_configuration, ...)
-   local new_app_graph = self.setup_fn(new_config)
-   local actions = self.support.compute_config_actions(
-      self.current_app_graph, new_app_graph, to_restart, verb, path, ...)
-   self:enqueue_config_actions(actions)
-   self.current_app_graph = new_app_graph
+   local new_graph = self.setup_fn(new_config, ...)
+   for id, follower in pairs(self.followers) do
+      local actions = self.support.compute_config_actions(
+         follower.graph, new_graph[id], to_restart, verb, path, ...)
+      self:enqueue_config_actions_for_follower(id, actions)
+      follower.graph = new_graph[id]
+   end
    self.current_configuration = new_config
-   self.current_in_place_dependencies =
+   self.in_place_dependencies =
```

Comment:
For me I would leave the `current_` moniker to match the `current_configuration`, but I dunno...